### ISSUE TITLE
`SIGTERM` hot-loop

### ISSUE THREAD

**Issue by clauseggers** [youridijk/homebridge-atag-one#2]
Great to see a plugin for the old ATAG One. I tried installing it, but Homebridge runs into a `SIGTERM` and restarts.
The Home app will on and off show the ATAG One, so I think your plugin finds the IP of the One, but some other problem occurs.

```
[2/13/2022, 3:08:22 PM] Homebridge v1.4.0 (HAP v0.10.0) (Homebridge 5264) is running on port 52808.
[2/13/2022,
3:08:22 PM] [Hue] Philips hue: event stream connected to https://192.168.1.9/eventstream/clip/v2
[2/13/2022, 3:08:22 PM] [Hue] warning: latest version: homebridge-hue v0.13.38
12/13/2022,
3:08:23 PM] [ATAG One] Error getting serial: Error: Base url is undefined!
[2/13/2022,
3:08:26 PM] Error: EACCES: permission denied, open
'device-config.json'
at Object.openSync (node: fs:585:3)
at Object.writeFileSync (node: fs:2155:35)
at updateDeviceConfig (/homebridge/node modules/homebridge-atag-one/src/files.ts:11:6)
at AtagOne.handleBroadcastMessage(/homebridge/nodemodules/homebridge-atag-one/src/AtagOne.ts:151:271
at
Socket.<anonymous> (/homebridge/node modules/homebridge-atag-one/src/AtagOne.ts:161:331
at
Socket.emit (node:events:520:28)
at
UDP.onMessage (node:dgram: 939: 8)
[2/13/2022,
3:08:26 PM] Got SIGTERM, shutting down Homebridge..
[2/13/2022,
3:08:31 PM] (Hue] goodbye
12/13/2022,
3:08:31 PM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
12/13/2022,
3:08:36 PM] [HB Supervisor] Restarting Homebridge...
[2/13/2022,
3:08:36 PM] [HB Supervisor] Starting Homebridge with extra flags:
-I
-P /homebridge/node modules
[2/13/2022,
3:08:36 PM] [HB Supervisor] Started Homebridge v1.4.0 with PID:
764
[2/13/2022,
3:08:37 PM] Loaded config.json with 1 accessories and 2 platforms.
[2/13/2022,
3:08:37 PM] Loaded 0 cached accessories from cachedAccessories.
12/13/2022,
3:08:37 PM1
==-
[2/13/2022,
3:08:38 PM] Loaded plugin: homebridge-atag-one@l.0.1
[2/13/2022,
3:08:38 PM] Registering accessory
'homebridge-atag-one.Atagone'
12/13/2022,
3:08:38 PM1
---
[2/13/2022,
3:08:38 PM] Loaded plugin: homebridge-dummy@0.6.0
[2/13/2022,
3:08:38 PM] Registering accessory
'homebridge-dummy.DummySwitch'
```

**Comment by youridijk** [comment]
The problem is that HomeBridge can't acces `device-config.json`.  This file is used to store the base url (IP-address of your Atag One + port 10000) of your Atag One between reboots of HomeBridge. 
Because there is no error catching for this error, HomeBridge crashes. And because the base url is stored in the first 10 seconds this plug-in starts up, HomeBridge keeps crashing. 

I fixed this error in the latest version, and in addition, I added an option to disable the storing of the base url.
